### DISCUSSION TITLE
UI not opening

### DISCUSSION THREAD

**Discussion by SKR6588** [Rem0o/FanControl.Releases#613] [Q&A] [answered]
I've been trying for weeks to get this amazing program to run, without success.
OS is up-to-date Win10.
The sensor discovery seems to work, the program runs and I can see it in the taskbar.
It's even right clickable, I can choose to open the UI but nothing happens.
I've tried everything to no avail. 

Can someone please help?

**Comment by xhul-dev** [comment, marked as answer]
Hello there.
Try this:

1) Right-click on the Fan Control systray icon.
2) Left-click on "Open".
3) When Fan Control appears in the taskbar, hover on it with mouse, then right-click on the peak that appears (preview mini-window).
4) Left-click on "Move".
5) Try moving the window using the arrow keys and/or mouse.

That should adjust the window coordinates so that it becomes visible.

**Reply by SKR6588** [reply]
Oh, that helped!
I tried clicking MAXIMIZE and there it went.
Thanks a bunch!
